tests/os/SpawnTest.php

```diff
@@ -0,0 +1,104 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+use type Facebook\HackTest\HackTest;
+use function HH\Lib\Experimental\OS\posix_spawnp;
```

Comment:
remove all `use function` - except for `expect` -  and `use type`, only `use namespace`